shell.nix

```diff
@@ -4,8 +4,8 @@
 }: with dappPkgs;
 
 mkShell {
-  DAPP_SOLC = solc-static-versions.solc_0_5_12 + "/bin/solc-0.5.12";
-  # SOLC_FLAGS = "--optimize --optimize-runs=200";
+  DAPP_SOLC = solc-static-versions.solc_0_6_12 + "/bin/solc-0.6.12";
+  SOLC_FLAGS = "--optimize --optimize-runs=1";
```

Comment:
Should we synchronize `optimize-runs` between Nix and the Makefile?

Reply:
Fixed! I think 1 run makes sense for this contract, right?

Reply:
Anyway someone burning 50K MKR doesn't really care about the gas costs...

Reply:
Yeah, I don't think more than once is really necessary...possibly `join` could receive multiple calls, but even that I have doubts about.